<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkoutDialog</class>
 <widget class="QDialog" name="WorkoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouvelle sortie</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>241</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="add_pushButton">
      <property name="text">
       <string>Valider</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources/ressources.qrc">
        <normaloff>:/bike/valid.png</normaloff>:/bike/valid.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbQuit">
      <property name="text">
       <string> Annuler</string>
      </property>
      <property name="icon">
       <iconset resource="../ressources/ressources.qrc">
        <normaloff>:/bike/cancel.png</normaloff>:/bike/cancel.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>581</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0,0">
    <item row="4" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Durée</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QSpinBox" name="spFCMax">
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Fréquence cardiaque moyenne</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Vitesse moyenne (km/h)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Distance (km)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QSpinBox" name="spFCMoyenne">
      <property name="maximum">
       <number>999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Fréquence caridiaque max</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="leNom">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Type de la sortie</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QDoubleSpinBox" name="spVitesseMoyenne">
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QComboBox" name="cbBike">
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nom de la sortie</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cbType">
      <item>
       <property name="text">
        <string>Route</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VTT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Piste</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Descente</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QSpinBox" name="spCadenceMax"/>
    </item>
    <item row="7" column="3">
     <widget class="QSpinBox" name="spPuissanceMax">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date de la sortie</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QTimeEdit" name="teSortie">
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Vélo utilisé</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QDateEdit" name="deSortie">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QDoubleSpinBox" name="spDistance">
      <property name="maximum">
       <double>99999.990000000005239</double>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Dénivelé (m)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Cadence moyenne</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Vitesse max (km/h)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="spEnergie">
      <property name="maximum">
       <number>9999999</number>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Energie (kJ)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="spDenivele">
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Puissance moyenne (w)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Cadence max</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Puissance max (w)</string>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QSpinBox" name="spCadenceMoyenne">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="spVitesseMax">
      <property name="maximum">
       <double>999.990000000000009</double>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="spPuissanceMoyenne">
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QRadioButton" name="rbSun">
      <property name="text">
       <string>Soleil</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QRadioButton" name="rbRain">
      <property name="text">
       <string>Pluie</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="rbWind">
      <property name="text">
       <string>Vent</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbImport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>148</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Importer GPX</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
